build-constraints.yaml

```diff
@@ -10,6 +10,9 @@ cabal-format-version: "3.0"
 # Constraints for brand new builds
 packages:
 
+    "David Burkett <david@davidburkett.xyz> @Disco-Dave>:
```

Comment:
```suggestion
    "David Burkett <david@davidburkett.xyz> @Disco-Dave":
```

Reply:
Yikes, sorry I missed that. Thanks for catching that!